plugins/providers/shield/client.go

```diff
@@ -145,13 +108,13 @@ func (c *client) GetAdminsOfGivenResourceType(ctx context.Context, id string, re
 	return userEmails, err
 }
 
-func (c *client) GetTeams(ctx context.Context) ([]*Team, error) {
+func (c *client) GetTeams(ctx context.Context) ([]*Group, error) {
 	req, err := c.newRequest(http.MethodGet, groupsEndpoint, nil, "")
 	if err != nil {
 		return nil, err
 	}
 
-	var teams []*Team
+	var teams []*Group
```

Comment:
Please also change the variable name to groups instead of teams here.

Reply:
ok